Mobl_ios_Callisto-master/MealTrackingPilot/ChatBot/ChatViewController.swift

```diff
@@ -196,9 +198,11 @@ class ChatViewController: UIViewController, UITableViewDataSource, UITableViewDe
         case .spacer:
             let cell = tableView.dequeueReusableCell(withIdentifier: SpacerCell.cellIdentifier, for: indexPath)
             cell.backgroundColor = UIColor.clear
+            cell.contentView.transform = CGAffineTransform(scaleX: 1, y: -1)
```

Comment:
Please use below for transform,
tableView.transform = CGAffineTransform(rotationAngle: (-.pi))